### PR TITLE
Prep 0.20.0rc0

### PR DESCRIPTION
**Pull request by jarrodmillman**
## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- Descriptive commit messages (see below)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
  `doc/release/release_dev.rst`.
- There is a bot to help automate backporting a PR to an older branch. For
  example, to backport to v0.19.x after merging, add the following in a PR
  comment: `@meeseeksdev backport to v0.19.x`
- To run benchmarks on a PR, add the `run-benchmark` label. To rerun, the label
  can be removed and then added again. The benchmark output can be checked in
  the "Actions" tab.

### CODE REVIEW

File: doc/release/release_0.20.rst [hunk 1]
```diff
@@ -0,0 +1,658 @@
+Announcement: scikit-image 0.20.0 (unreleased)
+==============================================
+
+We're happy to announce the release of scikit-image v0.20.0 (unreleased)!
```
Comment (stefanv) [inline doc/release/release_0.20.rst:4]: Should this be something like:

> We're happy to announce the pre-release of scikit-image v0.20.0 (rc1)!

?

File: doc/release/release_0.20.rst [hunk 2]
```diff
@@ -0,0 +1,658 @@
...
+
+scikit-image is an image processing toolbox built on SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+For more information, examples, and documentation, please visit our website:
+https://scikit-image.org
+
+With this release, many of the functions in ``skimage.measure`` now support anisotropic images  with different voxel spacings which can be specified with the new parameter ``spacing``.
```
Comment (stefanv) [inline doc/release/release_0.20.rst:13]: Is there a reason we don't wrap the release notes at 88?

Comment (stefanv) [inline doc/release/release_0.20.rst:13]:
```suggestion
With this release, many of the functions in ``skimage.measure`` now support anisotropic images with different voxel spacings which can be specified with the new parameter ``spacing``.
```

File: doc/release/release_0.20.rst [hunk 3]
```diff
@@ -0,0 +1,658 @@
...
+
+Support for footprint decomposition  was added to several functions in ``skimage.morphology``.
```
Comment (stefanv) [inline doc/release/release_0.20.rst:15]:
```suggestion
Support for footprint decomposition was added to several functions in ``skimage.morphology``.
```

File: doc/release/release_0.20.rst [hunk 4]
```diff
@@ -0,0 +1,658 @@
...
+By decomposing a footprint into several smaller ones with the new ``decomposition`` parameter, morphological operations can potentially be sped up.
```
Comment (stefanv) [inline doc/release/release_0.20.rst:16]:
```suggestion
By decomposing a footprint into several smaller ones with the new ``decomposition`` parameter, morphological operations can potentially be sped up significantly.
```

File: doc/release/release_0.20.rst [hunk 5]
```diff
@@ -0,0 +1,658 @@
...
+
+In addition a good number of performance improvements have been included.
+
+4 new gallery examples were added to our documentation; checkout the new interactive example "Track solidification of a metallic alloy".
```
Comment (stefanv) [inline doc/release/release_0.20.rst:20]: Worth adding a link?

File: doc/release/release_0.20.rst [hunk 6]
```diff
@@ -0,0 +1,658 @@
...
+
+This release also completes the transition to indicating multi-channel images with the more flexible ``channel_axis`` parameter as well as a many other deprecations to make scikit-image's API more consistent and expressive.
+
+This release now supports Python 3.8-3.11.
```
Comment (stefanv) [inline doc/release/release_0.20.rst:24]:
```suggestion
This release supports Python 3.8-3.11.
```

File: doc/release/release_0.20.rst [hunk 7]
```diff
@@ -0,0 +1,658 @@
...
+With this release, many of the functions in ``skimage.measure`` now support anisotropic images  with different voxel spacings which can be specified with the new parameter ``spacing``.
+
+Support for footprint decomposition  was added to several functions in ``skimage.morphology``.
+By decomposing a footprint into several smaller ones with the new ``decomposition`` parameter, morphological operations can potentially be sped up.
+
+In addition a good number of performance improvements have been included.
+
+4 new gallery examples were added to our documentation; checkout the new interactive example "Track solidification of a metallic alloy".
+
+This release also completes the transition to indicating multi-channel images with the more flexible ``channel_axis`` parameter as well as a many other deprecations to make scikit-image's API more consistent and expressive.
```
Comment (stefanv) [inline doc/release/release_0.20.rst:22]:
```suggestion
This release also completes the transition to the ``channel_axis`` parameter, a more flexible way of indicating multi-channel images, as well as many other deprecations that make scikit-image's API more consistent and expressive.
```

File: doc/release/release_0.20.rst [hunk 8]
```diff
@@ -0,0 +1,658 @@
...
+
+This release now supports Python 3.8-3.11.
+
+New features and improvements
+-----------------------------
+.. Anything new or improved - including performance - that is covered by our API.
+
```
Comment (stefanv) [inline doc/release/release_0.20.rst:29]:
```suggestion
```

File: doc/release/release_0.20.rst [hunk 9]
```diff
@@ -0,0 +1,658 @@
...
+  (`#6583 <https://github.com/scikit-image/scikit-image/pull/6583>`_).
+- Remove deprecated submodules ``skimage.morphology.grey`` and ``skimage.morphology.greyreconstruct``; use ``skimage.morphology`` instead
+  (`#6583 <https://github.com/scikit-image/scikit-image/pull/6583>`_).
+- Remove deprecated submodule ``skimage.morphology.selem``; use ``skimage.morphology.footprints`` instead
+  (`#6583 <https://github.com/scikit-image/scikit-image/pull/6583>`_).
+
+Bug fixes
+---------
+.. Only fixes to unexpected behavior in our public API. Exclude things such as fixes to the CI or documentation.
+
```
Comment (stefanv) [inline doc/release/release_0.20.rst:193]:
```suggestion
```

File: doc/release/release_0.20.rst [hunk 10]
```diff
@@ -0,0 +1,658 @@
...
+- Get rid of numpy deprecation warning in gallery example ``plot_equalize``
+  (`#6650 <https://github.com/scikit-image/scikit-image/pull/6650>`_).
+- Fix swapping of opening and closing in gallery example ``plot_rank_filters``
+  (`#6652 <https://github.com/scikit-image/scikit-image/pull/6652>`_).
+- Get rid of numpy deprecation warning in gallery example ``in plot_log_gamma.py``
+  (`#6655 <https://github.com/scikit-image/scikit-image/pull/6655>`_).
+
+Other and development related updates
+-------------------------------------
+.. Everything invisible to a user of scikit-image or not covered in the above section. These changes probably not interesting to users.
```
Comment (stefanv) [inline doc/release/release_0.20.rst:342]:
```suggestion
```

File: doc/release/release_0.20.rst [hunk 11]
```diff
@@ -0,0 +1,658 @@
...
+- Update imports/refs from deprecated scipy.ndimage.filters namespace (`#6231 <https://github.com/scikit-image/scikit-image/pull/6231>`_)
+- Include Cython sources via package_data (`#6232 <https://github.com/scikit-image/scikit-image/pull/6232>`_)
+- DOC: fix SciPy intersphinx (`#6239 <https://github.com/scikit-image/scikit-image/pull/6239>`_)
+- Fix bug in SLIC superpixels with `enforce_connectivity=True` and `start_label > 0` (`#6242 <https://github.com/scikit-image/scikit-image/pull/6242>`_)
+- Fowardport PR #6249 on branch main (update MacOS libomp installation in wheel building script) (`#6250 <https://github.com/scikit-image/scikit-image/pull/6250>`_)
+- Ignore sparse matrix deprecation warning (`#6261 <https://github.com/scikit-image/scikit-image/pull/6261>`_)
+- Add a textbook-like tutorial on measuring fluorescence at the nuclear envelope of a cell
+  (`#5262 <https://github.com/scikit-image/scikit-image/pull/5262>`_).
+
+66 authors added to this release [alphabetical by first name or login]
```
Comment (stefanv) [inline doc/release/release_0.20.rst:551]:
```suggestion
66 authors contributed to this release [alphabetical by first name or login]
```

contributed to? worked on?

File: doc/release/release_0.20.rst [hunk 12]
```diff
@@ -0,0 +1,658 @@
...
+- Ben Greiner
+- bsmietanka
+- Chris Roat
+- Chris Wood
+- Dave Mellert
+- Dudu Lasry
+- Elena Pascal
+- Eli Schwartz
+- Fabian Schneider
+- forgeRW
```
Comment (stefanv) [inline doc/release/release_0.20.rst:570]: For cases without real names, should we add `@` in front?

File: doc/release/release_0.20.rst [hunk 13]
```diff
@@ -0,0 +1,658 @@
...
+- Sanghyeok Hyun
+- Sebastian Wallkötter
+- Simon-Martin Schröder
+- Stefan van der Walt
+- Teemu Kumpumäki
+- Thomas Voigtmann
+- Tim-Oliver Buchholz
+- Tyler Reddy
+
+37 reviewers added to this release [alphabetical by first name or login]
```
Comment (stefanv) [inline doc/release/release_0.20.rst:620]:
```suggestion
37 reviewers worked on this release [alphabetical by first name or login]
```

File: doc/release/release_dev.rst [hunk 14]
```diff
@@ -1,658 +1,42 @@
...
-- Get rid of numpy deprecation warning in gallery example ``plot_equalize``
-  (`#6650 <https://github.com/scikit-image/scikit-image/pull/6650>`_).
-- Fix swapping of opening and closing in gallery example ``plot_rank_filters``
-  (`#6652 <https://github.com/scikit-image/scikit-image/pull/6652>`_).
-- Get rid of numpy deprecation warning in gallery example ``in plot_log_gamma.py``
-  (`#6655 <https://github.com/scikit-image/scikit-image/pull/6655>`_).
 
-Other and development related updates
--------------------------------------
-.. Everything invisible to a user of scikit-image or not covered in the above section. These changes probably not interesting to users.
```
Comment (stefanv) [inline doc/release/release_dev.rst:342]: Should probably keep these comments? You can maybe just grab the previous release_dev.rst template.

**Review by stefanv — APPROVED** [review]
Some minor things that we can address quickly.

File: doc/release/release_0.20.rst (hunk 10 above)
Comment (jarrodmillman) [reply doc/release/release_0.20.rst:347]: The remaining comments all relate to autogenerated text. I will address this in a future PR after 0.20.0rc0.

**Pull request merged**